### PR TITLE
Team 1 Week 1

### PR DESCRIPTION
**Pull request by minnon**

### CODE REVIEW

File: teams/Team1.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* UI for trading credits for ally units or repairs for player's ship
+* Lose when player's ship is destroyed, win when every zone is controlled
+
+## Advanced topics
+
+* Physics
+	* Gravity accurately simulated for all objects	
+	* Newton's laws of motion apply such that objects
+* Procedural generation
+	* Each zone dynamically generated as a solar system with varying objects and conditions
```
Comment (nfarnan) [inline teams/Team1.md]: What will the contents of the zones be? If they are mostly open space, procedural generation would be too simple.

File: teams/Team1.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* Free roam throughout the map's 7 zones
+* Credits increment at regular intervals with the amount depending on controlled zones and the objects in said zone
+* UI for trading credits for ally units or repairs for player's ship
+* Lose when player's ship is destroyed, win when every zone is controlled
+
+## Advanced topics
+
+* Physics
+	* Gravity accurately simulated for all objects	
+	* Newton's laws of motion apply such that objects
```
Comment (nfarnan) [inline teams/Team1.md]: What objects are you planning to have in the world?

File: teams/Team1.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* Objects all exist on same plane and can collide with each other
+* Free roam throughout the map's 7 zones
+* Credits increment at regular intervals with the amount depending on controlled zones and the objects in said zone
+* UI for trading credits for ally units or repairs for player's ship
+* Lose when player's ship is destroyed, win when every zone is controlled
+
+## Advanced topics
+
+* Physics
+	* Gravity accurately simulated for all objects
```
Comment (nfarnan) [inline teams/Team1.md]: Will objects be big enough for accurate gravity simulation to have a noticeable effect?

File: teams/Team1.md (hunk 1 above)
Comment (minnon) [reply teams/Team1.md]: Each zone would have at least one star and some number of planets which can have their own orbiting moons and differ in size and type. Many zones would have asteroids as well, which would be randomized to where they could collide with planets or other asteroids and spawn meteoroids.

File: teams/Team1.md (hunk 2 above)
Comment (minnon) [reply teams/Team1.md]: World objects will include stars, planets, moons, asteroids, meteoroids, space stations, and black holes.

File: teams/Team1.md (hunk 3 above)
Comment (minnon) [reply teams/Team1.md]: Stars, planets, and black holes will have noticeable effects. Moons, asteroids, space stations, and the largest class of ally/enemy unit will have some small effect as well. Everything else will just be feeling the pull of these things.

File: management/Team1.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Team 1 Management weeks
+
+Each team should copy and fill out this schedule:
```
[the remark this answers is not in the document]
Comment (minnon) [reply management/Team1.md]: Missed that one, but it would be 10/16 - 10/23.

[the remark this answers is not in the document]
Comment (nfarnan) [reply management/Team1.md]: Be sure to add that.

File: teams/Team1.md (hunk 3 above)
Comment (nfarnan) [reply teams/Team1.md]: This may need to be exaggerated gravity based on realistic models, but I can accept that.

File: teams/Team1.md (hunk 1 above)
Comment (nfarnan) [reply teams/Team1.md]: In this case, it seems that you are only generating spherical objects with a set size and orbit. Hence, you would only be generating a number of size, distance, and initial velocity values. This is a relatively simplistic random generation scheme. I do not see this being sufficiently challenging to count as an advanced topic.

**Pull request merged**